apps/devhub/tests/test_utils.py

```diff
@@ -0,0 +1,373 @@
+import os.path
+from copy import deepcopy
+
+import mock
+import pytest
+from mock import call
+
+from django.conf import settings
+
+import amo
+import amo.tests
+from addons.models import Addon
+from files.models import File, FileUpload
+from versions.models import Version
+from devhub import utils
+from devhub.tasks import annotate_validation_results
+
+
+pytestmark = pytest.mark.django_db
+
+
+def change_dict(src, changes):
+    src = src.copy()
+    src.update(changes)
+    return src
+
+
+class TestValidationComparator(amo.tests.TestCase):
+    SIGNING_SUMMARY = {'high': 0, 'medium': 0, 'low': 0, 'trivial': 0}
+
+    def compare(self, old, new, expected):
+        """Compare two messages, and assert that the expected annotations are
+        present.
+
+        `old` is a message dict from the previous set of results,
+        `new` is a dict containing the set of property changes between
+        the old message and the new one, and `expected` is the full set of
+        annotations expected to be added to the new message."""
```

Comment:
This is a bit confusing. So `new` is only the set of changes? Maybe rename to `changes` or `diff`?